cmd/spicedb/schemawatch_integration_test.go

```diff
@@ -0,0 +1,127 @@
+//go:build docker && image
+// +build docker,image
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+	"github.com/stretchr/testify/require"
+
+	testdatastore "github.com/authzed/spicedb/internal/testserver/datastore"
+	"github.com/authzed/spicedb/pkg/datastore"
+)
+
+func TestSchemaWatch(t *testing.T) {
+	engines := map[string]bool{
+		"postgres":    false,
+		"mysql":       false,
+		"cockroachdb": true,
+		"spanner":     false,
+	}
+	require.Equal(t, len(engines), len(datastore.Engines))
+
+	for driverName, shouldRun := range engines {
+		if !shouldRun {
+			continue
+		}
+
+		t.Run(driverName, func(t *testing.T) {
+			bridgeNetworkName := fmt.Sprintf("bridge-%s", uuid.New().String())
```

Comment:
seem like expensive tests, let's parallelize them

```suggestion
			t.Parallel()
			
			bridgeNetworkName := fmt.Sprintf("bridge-%s", uuid.New().String())
```

Reply:
I don't like running datastore container tests in parallel; things to get wonky when we do that